test/resources/resourceConfig.spec.js

```diff
@@ -9,7 +9,7 @@ describe('Build endpoint', () => {
     const endpoint = 'http://new.shoutem.com';
     const params = { test: 1 };
 
-    const expectedEndpoint = 'http://new.shoutem.com';
+    const expectedEndpoint = 'http://new.shoutem.com/?test=1';
```

Comment:
I am not sure this is the expected endpoint here:
- if you look at the test name, no mention of query params, this is just a basic endpoint test.
- query params are not listed in the `endpoint` part and thus, are not expected in the resulting string.

(check below for the `with query params` & `with query params not provided` tests to see what I mean)

Reply:
Did you rerun tests?
This one should now be reverted (by default - unused params should be ignored).